packages/adminrouter/windows.buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "sources": {
         "OpenResty": {
           "kind": "url",
-          "url": "https://openresty.org/download/openresty-1.15.8.2-win64.zip",
-          "sha1": "8379fdcb93f1bb6ace8b9f575c5fe60b831182fc"
+          "url": "https://openresty.org/download/openresty-1.15.8.3-win64.zip",
```

Comment:
We should host this in our own s3 artifactory.

Reply:
Sure, but I suggest that we merge this now to get merging unblocked.